languages/cpp/exercises/concept/enums/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "SourangshuGhosh,
+      "exercism_username": "SourangshuGhosh"
+    }
+  ],
+  "forked_from": ["exercism/v3"]
```

Comment:
The "exercism/v3" value is not a valid value for the `"forked_from"` field. Instead, it should point to the exercise that is being forked. See [the docs](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md#metaconfigjson).